setup.py

```diff
@@ -81,6 +82,28 @@ def clean_optional_plugins():
         walk_plugins_setup_py(action_name="CLEANING", action=["clean", "--all"])
 
 
+class Develop(setuptools.command.develop.develop):
+    """Custom develop command."""
+
+    def run(self):
+
+        # python setup.py develop --user --uninstall
+        # we uninstall the plugins before uninstalling avocado
+        if self.user and not self.uninstall:
+            setuptools.command.develop.develop.run(self)
+            walk_plugins_setup_py(action_name="LINK", action=["develop", "--user"])
+
+        # python setup.py develop --user
+        # we install the plugins after installing avocado
+        elif self.user and self.uninstall:
+            walk_plugins_setup_py(action_name="UNLINK", action=["develop", "--uninstall", "--user"])
+            setuptools.command.develop.develop.run(self)
+
+        # other cases: do nothing and call parent function
+        else:
+            setuptools.command.develop.develop.run(self)
```

Comment:
`super().run()`

Reply:
Thanks, I have updated this and the others two.